<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerGraphCutSegmentationModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerGraphCutSegmentationModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Input Volumes </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="qMRMLNodeComboBox" name="objectMRMLNodeComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Object Volume:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="inputMRMLNodeComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Input Volume:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Background Volume</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="qMRMLNodeComboBox" name="backgroundMRMLNodeComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="startPushButton">
        <property name="text">
         <string>Start Segmentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>startPushButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerGraphCutSegmentationModuleWidget</receiver>
   <slot>segment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerGraphCutSegmentationModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerGraphCutSegmentationModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>backgroundMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerGraphCutSegmentationModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>objectMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>segment()</slot>
 </slots>
</ui>
